#
# base patch: meta/recipes-core/sysfsutils/sysfsutils-2.1.0/obsolete_automake_macros.patch
# base branch: master
# base commit: dbda297fd91aab2727f7a69d3b7d3a32ad4261d2
#
# Upstream-Status: Submitted [http://sourceforge.net/tracker/?func=detail&aid=3600345&group_id=44427&atid=439544]
#
# Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
diff -Nurd sysfsutils-2.1.0/configure.ac sysfsutils-2.1.0/configure.ac
--- sysfsutils-2.1.0/configure.ac	2006-08-07 08:08:00.000000000 +0300
+++ sysfsutils-2.1.0/configure.ac	2013-01-11 08:13:08.651550634 +0200
@@ -2,7 +2,7 @@
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([config.h.in])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])

 # Checks for KLIBC support (should be before AC_PROG_LIBTOOL and AC_PROG_CC)
 AC_CHECK_KLIBC
